pkg/apis/enterprise/v2/common_types.go

```diff
@@ -71,24 +71,6 @@ const (
 	DeployStatusError
 )
 
-// AppDownloadState represents the download status of app
```

Comment:
This is now rolled into the app deployment status, correct?

Reply:
Yeah as part of PhaseInfo